native/packages/react-native-backpack/libraries/Panel/android/native/src/main/java/net/skyscanner/backpack/BpkPanel.kt

```diff
@@ -0,0 +1,50 @@
+package net.skyscanner.backpack
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.LinearLayout
+
+
+class BpkPanel : LinearLayout {
+
+    private var mPadding: Boolean = false
+
+    constructor(context: Context) : super(context) {
+        this.orientation = LinearLayout.VERTICAL
+    }
+
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, R.style.Bpk_panel) {
```

Comment:
You're receiving `defStyleAttr` in the constructor but passing `R.style.Bpk_panel` to the super class